src/ucp/dt/datatype_iter.inl

```diff
@@ -334,33 +355,70 @@ ucp_datatype_iter_is_end(const ucp_datatype_iter_t *dt_iter)
     return dt_iter->offset == dt_iter->length;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_status_t ucp_datatype_iter_mem_reg_internal(
+        ucp_context_h context, ucp_datatype_iter_t *dt_iter,
+        ucp_md_map_t md_map, void *buffer, size_t length, unsigned uct_flags,
+        ucp_dt_reg_t *dt_reg)
```

Comment:
```suggestion
static UCS_F_ALWAYS_INLINE ucs_status_t
ucp_datatype_iter_mem_reg_internal(ucp_context_h context,
                                   ucp_datatype_iter_t *dt_iter,
                                   ucp_md_map_t md_map, void *buffer,
                                   size_t length, unsigned uct_flags,
                                   ucp_dt_reg_t *dt_reg)
```

Reply:
the current format was done by clang-format, will change